plugins/tactics/test/AutoTupleSpec.hs

```diff
@@ -48,10 +48,11 @@ spec = describe "auto for tuple" $ do
             (auto' $ n * 2) `shouldSatisfy` isRight
 
 
+{- HLINT ignore randomGroups "Redundant <$>"-}
+
 randomGroups :: [a] -> Gen [[a]]
 randomGroups [] = pure []
 randomGroups as = do
   n <- choose (1, length as)
   (:) <$> pure (take n as)
       <*> randomGroups (drop n as)
```

Comment:
plugins/tactics/test/AutoTupleSpec.hs:55:3-26: Warning: Redundant <$>
Found:
  (:) <$> pure (take n as)
      <*> randomGroups (drop n as)
Perhaps:
  pure ((:) (take n as))
      <*> randomGroups (drop n as)

Not an improvement.

Reply:
Wouldn’t `(take n as:) <$> randomGroups (drop n as)` work?

Reply:
Changed, thanks!